integrated_channels/api/v1/logs/serializers.py

```diff
@@ -66,6 +66,33 @@ class Meta:
         model = LearnerDataTransmissionAudit
         fields = '__all__'
 
+    sync_status = serializers.SerializerMethodField()
+    sync_last_attempted_at = serializers.SerializerMethodField()
+
+    def get_sync_status(self, obj):
+        """
+        Return a string representation of the sync status.
+        """
+        if obj.status is None:
+            sync_status = 'pending'
+        elif type(obj.status) is not int or int(obj.status) >= 400:
+            sync_status = 'error'
+        elif int(obj.status) < 400:
+            sync_status = 'okay'
+
+        return sync_status
+
+    def get_sync_last_attempted_at(self, obj):
+        """
+        Return the most recent/youngest sync attempt date.
+        """
+        date_list = [obj.created, obj.modified]
+        res = [i for i in date_list if i is not None]
+        if not res:
+            return None
+        else:
+            return max(res)
```

Comment:
nit: Is there a possibility that an object's created date would be later than its modified date?  If not, you could replace this with 
```suggestion
    return obj.modified or obj.created
```

Reply:
hmmmmm yes I believe you're right!